tests/www/dashboard/test_edit_user_info.py

```diff
@@ -472,28 +472,7 @@ def test_edit_with_invalid_pole_emploi_id(self):
         )
 
     def test_edit_as_prescriber(self):
-        user = PrescriberFactory()
-        self.client.force_login(user)
-        url = reverse("dashboard:edit_user_info")
-        response = self.client.get(url)
-        self.assertNotContains(response, self.NIR_FIELD_ID)
-        self.assertNotContains(response, self.LACK_OF_NIR_FIELD_ID)
-        self.assertNotContains(response, self.LACK_OF_NIR_REASON_FIELD_ID)
-        self.assertNotContains(response, self.BIRTHDATE_FIELD_NAME)
-
-        post_data = {
-            "first_name": "Bob",
-            "last_name": "Saint Clar",
-            "phone": "0610203050",
-        }
-        response = self.client.post(url, data=post_data)
-        assert response.status_code == 302
-
-        user = User.objects.get(id=user.id)
-        assert user.phone == post_data["phone"]
-
-    def test_edit_as_prescriber_with_ic(self):
-        user = PrescriberFactory(identity_provider=IdentityProvider.INCLUSION_CONNECT)
+        user = PrescriberFactory(for_snapshot=True)
```

Comment:
Je préférerais `first_name="Not Bob",` (et consorts) plutôt que `for_snapshot` vu qu'on ne fait pas de snapshot :grimacing: